Logs don't work when usin main.py
